tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["tslint:latest", "tslint-react"],
   "rules": {
+    "ban": [true, ["Object", "assign"]],
```

Comment:
You can provide explanations directly in the rule config https://palantir.github.io/tslint/rules/ban/